example/storybook/stories/TrackTile/TrackTiles.stories.tsx

```diff
@@ -5,65 +5,41 @@ import { MockEnvironmentDecorator } from './util/MockEnvironmentDecorator';
 import { action } from '@storybook/addon-actions';
 import { TrackTile } from '../../../../src/components/TrackTile';
 import { CenterView } from '../../helpers/CenterView';
+import { getTrackers } from './util/trackerData';
+import { boolean } from '@storybook/addon-knobs';
+import { PillarsTile } from '../../../../src/components/TrackTile/PillarsTile/PillarsTile';
 
-storiesOf('TrackTile', module)
+const trackers = [...getTrackers(), ...getTrackers({ pillars: true })];
+
+storiesOf('TrackTile/TrackTiles', module)
   .addDecorator((storyFn, context) =>
     MockEnvironmentDecorator({
-      trackers: [
-        {
-          id: '1',
-          metricId: '1',
-          name: 'Test',
-          color: 'chartreuse',
-          resourceType: 'Procedure',
-          units: [
-            {
-              code: 'h',
-              default: true,
-              display: 'hour',
-              system: 'http://unitsofmeasure.org',
-              target: 5,
-              unit: 'h',
-            },
-          ],
-        },
-      ],
+      trackers,
     })(storyFn, context),
```

Comment:
This wasn't working (no trackers were being shown) because it didn't have `installed: true`.   Reused the pillars tracker data and fixed it there.